pkg/registry/controller/etcd/etcd.go

```diff
@@ -163,12 +163,13 @@ func (r *ScaleREST) Update(ctx api.Context, obj runtime.Object) (runtime.Object,
 	if err != nil {
 		return nil, false, err
 	}
-	return scaleFromRC(rc), false, nil
+	return ScaleFromRC(rc), false, nil
 }
 
-// scaleFromRC returns a scale subresource for a replication controller.
-func scaleFromRC(rc *api.ReplicationController) *autoscaling.Scale {
-	return &autoscaling.Scale{
+// ScaleFromRC returns a scale subresource for a replication controller.
+func ScaleFromRC(rc *api.ReplicationController) *extensions.Scale {
```

Comment:
We'll need 2 copies of this. 1 here, for autoscaling.Scale, and another in experimental/controller, with extensions.Scale